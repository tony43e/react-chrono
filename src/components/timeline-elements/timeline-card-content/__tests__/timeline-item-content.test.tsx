import { render } from '@testing-library/react';
import React from 'react';
import Content from '../timeline-card-content';

const fn = jest.fn();

test('Test Timeline Item Content', () => {
  const { getByText } = render(
    <Content content="This is a test" onShowMore={fn} />,
  );
  const contentElement = getByText(/test/i);
  expect(contentElement).toBeInTheDocument();
});

test('Test Timeline Item Content Snapshot', () => {
  const { getByText } = render(
    <Content content="This is a test" onShowMore={fn} />,
  );
  const contentElement = getByText(/test/i);
  expect(contentElement).toMatchSnapshot();
});