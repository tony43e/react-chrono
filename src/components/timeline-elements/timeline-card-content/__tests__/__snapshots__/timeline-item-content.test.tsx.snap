// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Timeline Item Content Snapshot 1`] = `
<p
  class="card-sub-title css-4qf4pr"
>
  This is a test
</p>
`;
